### DISCUSSION TITLE
404 file not found

### DISCUSSION THREAD

**Discussion by middlepath79** [community/community#22698] [Other Feature Feedback, Questions, & Ideas]
<p><a href="http://Middlepath79.github.io" rel="nofollow noopener">Middlepath79.github.io</a> please can someone please check I have tried everything I can</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/85635fde722806398977df1758692f47c4ae4113.jpeg" data-download-href="/uploads/short-url/j20pzndSLly8PPVNrqPFs2266wb.jpeg?dl=1" title="1597763345456901652697348621801" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/58609915/181072632-afae908f-8bd4-49f7-88af-03cd9f0fb14b.jpeg" alt="1597763345456901652697348621801" data-base62-sha1="j20pzndSLly8PPVNrqPFs2266wb" width="375" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/85635fde722806398977df1758692f47c4ae4113_2_375x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/2X/8/85635fde722806398977df1758692f47c4ae4113_2_562x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/85635fde722806398977df1758692f47c4ae4113_2_750x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/85635fde722806398977df1758692f47c4ae4113_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">1597763345456901652697348621801</span><span class="informations">1920×2560 1020 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/middlepath79">@middlepath79</a>, welcome to the GitHub Support Community!</p>
<p>Your website is currently returning a 404 error, because the index.html is incorrectly named. Rename your <strong>INdex.html</strong> to <strong>index.html</strong> and it should resolve the 404 error. The filename is case-sensitive, so you have to make sure that index.html is written in lowercase characters.</p>
<p>If after renaming the file you’re still getting a 404 error. Wait at least 10 minutes for the page to be updated server-side or try clearing your browser cache/cookies.</p>

**Comment by middlepath79** [comment]
<p>Thanks :blush: problem resolved</p>

**Comment by Nagavi556** [comment]
![Screenshot 2025-06-10 164908](https://github.com/user-attachments/assets/53c4f726-117b-4b1d-9394-3b0abf0a9c53)
please help me out

**Reply by ngannguyen03081979-stack** [reply]
tại sao tôi không vào được
